user-guide/Advanced_Modules/DOM/DOM_ModuleSettings/DOM_StorageSettings.md

```diff
@@ -0,0 +1,23 @@
+---
+uid: DOM_StorageSettings
+---
+
+# StorageSettings
+
+The `StorageSettings` currently contains the configuration to define the caching behavior in the DOM manager. This is the only configurable option available in terms of storage at the moment.
+
+## CachingSettings
+
+The `CachingSettings` can be used to define whether or not the configuration DOM objects should be cached. This settings object contains the following properties:
+
+|Property                           |Type                   |Description |
+|-----------------------------------|-----------------------|------------|
+|DomDefinitionCachingPolicy         |DomConfigCachingPolicy |Caching policy for the `DomDefinition` objects. |
+|DomBehaviorDefinitionCachingPolicy |DomConfigCachingPolicy |Caching policy for the `DomBehaviorDefinition` objects. |
+|SectionDefinitionCachingPolicy     |DomConfigCachingPolicy |Caching policy for the `SectionDefinition` objects. |
+
+For each type, you can define one of three caching policy options:
+
+- **Default**: The default option. This will be selected for any new and existing managers. Behaves the same way as the "Full" option.
```

Comment:
"any new and existing managers"
=> This is a little confusing, and might need to be elaborated a bit. Do you mean that it is automatically selected for existing managers when you upgrade to 10.3.9/10.4.0 (assuming that this is the version introducing this)?

Reply:
From version X onward, any DomManager that was created before that, will use the caching. Any new manager will also use the caching by default if you do not specifically disable it when creating that manager. You can disable the cache on existing mangers by manually updating the new option on the ModuleSettings. Is this more clear? How would you suggest that I phrase it for the docs?

Reply:
How about this? I've phrased it so that it doesn't matter that I don't know the DataMiner version yet:

```suggestion
- **Default**: The default option. This will be automatically selected for new managers and for any managers that already existed before this feature was introduced. Behaves the same way as the "Full" option.
```